Lib/test/test_pathlib.py

```diff
@@ -2312,6 +2312,21 @@ def test_concrete_class(self):
         self.assertIs(type(p),
             pathlib.WindowsPath if os.name == 'nt' else pathlib.PosixPath)
 
+    def test_kwargs(self):
+        with self.assertRaisesRegex(TypeError, 'got an unexpected keyword argument'):
+            self.cls(arg=None)
+
+    def test_subclass_kwargs(self):
```

Comment:
`PosixPath` and `Windows Path` are subclasses of `Path` so I think you could move `test_kwargs` in `_BasePathTest` and you would not need `test_subclass_kwargs` anymore as it would test `Path`, `PosixPath` and `WindowsPath`.

Reply:
The main idea of `test_subclass_kwargs` is to test that you can subclass `Path` class and add custom `__init__` method which can accept `**kwargs`. This case was mentioned at bug discussion, so I thought that it will be a great idea to add a test to cover such case.

I totally agree with moving `test_kwargs` to `_BasePathTest`, I didn't notice that such class exists, my bad. 

What do you think about `test_subclass_kwargs`, should I remove it?

Reply:
Oh no, you are right a test for a custom `__init__` is needed, you should keep it.

Reply:
Done, I move tests to `_BasePathTest` class.